custom/icds_reports/management/commands/remove_duplicates.py

```diff
@@ -0,0 +1,108 @@
+from datetime import date
+
+import time
+from django.core.management.base import BaseCommand
+
+from django.db import connections, transaction
+from corehq.apps.userreports.util import get_table_name
+from corehq.sql_db.connections import get_icds_ucr_citus_db_alias
+from corehq.apps.userreports.tasks import _get_config_by_id, _build_indicators
+from corehq.apps.change_feed.data_sources import get_document_store_for_doc_type
+from pillowtop.dao.couch import ID_CHUNK_SIZE
+
+@transaction.atomic
+def _run_custom_sql_script(command):
+    db_alias = get_icds_ucr_citus_db_alias()
+    if not db_alias:
+        return
+
+    with connections[db_alias].cursor() as cursor:
+        cursor.execute(command)
+
+    return cursor.fetchall()
+
+class Command(BaseCommand):
+    DOMAIN_NAME = 'icds-dashboard-qa'
+    UCR_NAMES = [
+        'static-child_health_cases',
+        'static-ccs_record_cases',
+        'static-person_cases_v3',
+        'static-household_cases',
+        'static-child_tasks_cases',
+        'static-pregnant-tasks_cases',
+    ]
+
+    def add_arguments(self, parser):
+        parser.add_argument('ucr_name')
+
+    def handle(self, ucr_name, *args, **options):
+
+        for ucr in self.UCR_NAMES:
+            if ucr != 'ucr_name':
+                continue
```

Comment:
what's this?

Reply:
typo it was it should be `ucr != ucr_name`

Reply:
fixed.